cypress/integration/login.js

```diff
@@ -9,16 +10,18 @@ describe('Login functionality', function() {
 
     it('Should be able to login with role User', function () {
         cy.get('[id="loginForm.userId"]').click({force:true})
-        cy.get('[aria-label="Demo User"]').click()
+        cy.get('[aria-label="TestCon User 26"]').click()
         cy.get('[id="loginForm.role"]').click({force:true})
-        cy.get('[aria-label="User"]').click()
+        cy.get('[aria-label="Team Lead"]').click()
         cy.get('[type="submit"]').click()
 
         cy.url().should('include', '/time-logging')
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User')
-        cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('.user-info__title').contains('TestCon User 26')
+        cy.get('.main-nav').find('li').should('have.length', 2)
+        cy.get('.calendar--selected').should('have.class', 'calendar--today')
+        cy.get('.calendar--today').find('span').contains(d.getDate())
```

Comment:
".find('span')." is not required in this case, and this adds element chaining thus this locator might be more fragile with future web application updates. But this is not a blocker for pull request